### DISCUSSION TITLE
Is there an upgrade guide for 2.8 to 2.9? It's not in the upgrade docs

### DISCUSSION THREAD

**Discussion by nickjj** [argoproj/argo-cd#16297] [Q&A]
Hi,

I noticed [2.9.0 came out recently](https://github.com/argoproj/argo-cd/releases/tag/v2.9.0) which links to the upgrade guide at https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/overview/ but there isn't an entry for 2.8 to 2.9.

Does that mean there's zero manual steps or considerations to go from 2.8.X to 2.9.0 as a drop in upgrade?

**Comment by fmbahrt** [comment]
You can perhaps look here https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/upgrading/2.8-2.9.md but I don't know if it is complete.

**Reply by nickjj** [reply]
Thanks. I'll wait a bit to see if anything pops up before upgrading. I also see 2.9.1 shipped but the guide is also not published.

**Comment by Mais316** [comment]
Can anyone advise here, I am planning to upgrade from 2.7 to 2.9 but docs are not really clear and I see many changes in release notes of 2.9

Please advise.

**Comment by PhamQuang-512** [comment]
have you find any answer?

**Reply by crenshaw-dev** [reply]
The notes are here: https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.8-2.9/
